tutorindigo/plugin.py

```diff
@@ -102,68 +105,27 @@ def _override_openedx_docker_image(
 hooks.Filters.CONFIG_OVERRIDES.add_items(list(config["overrides"].items()))
 
 
+#  MFEs that are styled using Indigo
+indigo_styled_mfes = ['learning', 'learner-dashboard', 'profile', 'account', 'discussions']
+
 hooks.Filters.ENV_PATCHES.add_items(
-    [
```

Comment:
Does this change affect in any way the customisation instructions from the README? (In particular the troubleshooting section)

Reply:
Yes, I'll add once the basic functionality will work (with tutor-mfe PR)